ui/admin/app/routes/scopes/scope/scopes/index.js

```diff
@@ -29,6 +30,20 @@ export default class ScopesScopeScopesIndexRoute extends Route {
 
   // =methods
 
+  /**
+   * Event to determine whether the loading template should be shown.
+   * Only show the loading template during initial loads or when transitioning
+   * from different routes. Don't show it when a user is just searching or
+   * filtering on the same page as it can be jarring.
+   * @param transition
+   * @returns {boolean}
+   */
+  @action
+  loading(transition) {
+    const from = transition.from?.name;
+    return from !== 'scopes.scope.scopes.index';
```

Comment:
I think this would prevent transitions from orgs to projs or any scope list to another scope list really from showing a loading indicator since they're technically the same route?

Reply:
Oh great point. Let me test this a bit more.

Reply:
Not exactly sure why yet but we don't need this logic on any of the routes as I thought. The loading screen is shown upon initial entry to the specific routes but when searching or filtering, the loading screen is not getting triggered. This is the desired behavior.